lib/RegexTools.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var RegexTools = {};
```

Comment:
There's no reason for this object to exist, just have each "public" function be it's own file.  In this case `getGoogleGroupLinks.js` should be all you need.